nxc/protocols/ftp.py

```diff
@@ -14,11 +14,12 @@ def __init__(self, args, db, host):
         super().__init__(args, db, host)
 
     def proto_logger(self):
+        self.port = self.args.port
```

Comment:
this should probably be set in `__init__` of the `connection.py` and not in the protocol itself, same for the other protocols

Reply:
Fixed!